homeassistant/config_entries.py

```diff
@@ -245,6 +246,9 @@ def __init__(self, version: str, domain: str, title: str, data: dict,
         # Config data
         self.data = data
 
+        # Entry options
+        self.options = options
```

Comment:
I think that we should make sure that the options are always an empty dict at least, never `None`.